src/frontend/screens/Onboarding/JoinProjectQrScreen.tsx

```diff
@@ -26,14 +27,41 @@ const m = defineMessages({
     id: "screens.Onboarding.JoinProjectQrScreen.instructionsDescription",
     defaultMessage: "Show this QR code to your Project Admin",
   },
+  instructionDescriptionAsAdmin: {
+    id: "screens.Onboarding.JoinProjectQrScreen.instructionDescriptionAsAdmin",
+    defaultMessage: "Have the Project Participant scan this invite code.",
+  },
   sendJoinRequest: {
     id: "screens.Onboarding.JoinProjectQrScreen.sendJoinRequest",
     defaultMessage: "Send Join Request instead",
   },
+  cancel: {
+    id: "screens.Onboarding.JoinProjectQrScreen.cancel",
+    defaultMessage: "Cancel",
+  },
+});
+
+//Cancels the QR code screen and resets the navigation stack so the
+//user CANNOT hit the back button to go back to this screen
+const resetAction = StackActions.reset({
```

Comment:
curious but does `StackActions.pop` not do this? just from the name i'd imagine it would remove it from the stack and therefore be unable to navigate back to it via the back button, but not super familiar with the details of the API

Reply:
`pop` and `back` do the same thing. So `pop` will just take the current screen of the stack, bringing you back to the previous screen. But in this case, I want the user to go back home AND for the stack to completly reset